password/impl/src/test/java/org/wildfly/security/password/impl/MaskedPasswordTest.java

```diff
@@ -79,6 +80,9 @@ public static void removeProvider() {
 
     @Test
     public void testClearSpec() throws Exception {
+        Assume.assumeFalse("IBM JDK does not support some of the algorithms used in this test. See ELY-2063.",
```

Comment:
Is necessary to ignore all algorithms on IBM JDK in this test? Many of them are working.

Reply:
Good point, we should make sure that we only ignore the unsupported algorithms when running on IBM JDK.